### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by sreevatsanraman**

### CODE REVIEW

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 1]
```diff
@@ -31,6 +31,51 @@ Cask Data Application Platform Release Notes
...
+Improvements
+------------
+
+- :cask-issue:`CDAP-13116` - Macro enabled all fields in the HTTP Callback plugin
+
+- :cask-issue:`CDAP-13119` - Optimized the planner to reduce the amount of temporary data required in certain types of mapreduce pipelines. 
+
+- :cask-issue:`CDAP-13122` - Minor optimization to reduce the number of mappers used to read intermediate data in mapreduce pipelines
+
+- :cask-issue:`CDAP-13139` - Improves the schema generation fro database sources.
```
Comment (bdmogal) [inline cdap-docs/reference-manual/source/release-notes.rst]: typo: for

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 2]
```diff
@@ -31,6 +31,51 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-13179` - Automatic restart of long running program types (Service and Flow) upon application master process failure in YARN
```
Comment (bdmogal) [inline cdap-docs/reference-manual/source/release-notes.rst]: Seems like an incomplete statement? `Automatically restart long running...`?

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 3]
```diff
@@ -31,6 +31,51 @@ Cask Data Application Platform Release Notes
...
+Bug Fixes
+---------
+
+- :cask-issue:`CDAP-12875` - Fixed a bug that caused errors in the File source if it read parquet files that were not generated through Hadoop.
+
+- :cask-issue:`CDAP-13110` - Fixes an issue where a dataset's class loader was closed before the dataset itself, preventing the dataset from closing properly.
+
+- :cask-issue:`CDAP-13120` - Fixed a bug that caused directories to be left around if a workflow used a partitioned fileset as a local dataset
+
+- :cask-issue:`CDAP-13123` - Fixed a bug that stream hive exploration query does not work in an impersonated namespace.
```
Comment (bdmogal) [inline cdap-docs/reference-manual/source/release-notes.rst]: `Fixed a bug that caused a hive Explore query on Streams to not work...`

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 4]
```diff
@@ -31,6 +31,51 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-13129` - Fixed a planner bug to ensure that sinks are never placed in two different mapreduce phases in the same pipeline.
+
+- :cask-issue:`CDAP-13158` - Fixed a race condition when running multiple spark programs concurrently at a Workflow fork that can lead to workflow failure
+
+- :cask-issue:`CDAP-13171` - Fixes an issue with creating a namespace if the namespace principal is not a member of the namespace home's group.
```
Comment (bdmogal) [inline cdap-docs/reference-manual/source/release-notes.rst]: s/Fixes/Fixed/ to make it consistent?

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 5]
```diff
@@ -31,6 +31,51 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-13191` - Fixed a bug that caused completed run records to be missed when storing run state, resulting in misleading log messages about ignoring killed states.
+
+- :cask-issue:`CDAP-13192` - Fixes a bug in FileBatchSource that prevented ignoreFolders property from working with avro and parquet inputs
```
Comment (bdmogal) [inline cdap-docs/reference-manual/source/release-notes.rst]: ditto

**Review by bdmogal — COMMENTED** [review]
Minor comments. Rest LGTM

**Comment by sreevatsanraman** [comment]
Fixed comments. Merging.

**Pull request merged**